native/c/ztype.h

```diff
@@ -74,6 +74,22 @@ typedef struct
 
 } ZDIAG;
 
+typedef struct CamlstSearchParams
```

Comment:
i think these two structs make more sense in `zdstype.h`, most of `ztype.h` are generically common definitions